django/contrib/staticfiles/storage.py

```diff
@@ -51,6 +51,10 @@ class HashedFilesMixin:
                 r"""(?P<matched>@import\s*["']\s*(?P<url>.*?)["'])""",
                 """@import url("%(url)s")""",
             ),
+            (
+                r'(?m)(?P<matched>)^(/\*# (?-i:sourceMappingURL)=(?P<url>.*) \*/)$',
```

Comment:
I considered supporting varying amounts of whitespace before/after `sourceMappingURL` but decided not to since the spec doesn't mention it.